bkrdoc/analysis/generator/documentation_generator.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/python
 # author Jiri Kulda
-import bkrdoc
+
 import os
 # from __future__ import division  # floating-point division
 import argparse
 
-from bkrdoc.analysis.parser.bkrdoc_parser import Parser
-from bkrdoc.analysis.generator.credibility import DocumentationCredibility
+from bkrdoc.analysis.parser import bkrdoc_parser, conditions_for_commands
```

Comment:
From previous commit you rewritten all imports to from something import class. But now in this commit you are coming back to the old version. Choose one style and stick with it. Make it consistent.